skills/stash-supabase/SKILL.md

```
@@ -548,13 +552,15 @@ export const users = encryptedTable('users', {
 })
 ```
 
-Re-push the encryption config so EQL has a pending row that points at `email` (no `_encrypted` suffix):
+> **Known gap (SDK-only users):** `stash encrypt cutover` currently requires a pending EQL configuration, which is set by `stash db push`. If you're using the SDK without Proxy, you'll hit a "No pending EQL configuration" error from cutover. **Workaround:** run `stash db push` once before `stash encrypt cutover`. This will be decoupled in a future release — see [issue #447](https://github.com/cipherstash/stack/issues/447).
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)